packages/terra-date-picker/tests/jest/CalendarFilterr.test.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Whoops this file is name `CalendarFilterr` instead of `CalendarFilter`

Reply:
🙆🏻‍♂️ changed it : [30937dcf382](https://github.com/cerner/terra-framework/pull/1134/commits/8ec0355f75cba2e79d27fbd326b343bec97afba4#diff-6f43a0a00eb186c91a44a30937dcf382)